tests/integration/hub_usage/dummyhub/requirements.txt

```diff
@@ -0,0 +1,2 @@
+jina
```

Comment:
do not use `jina` in any of our own CICD, this pollutes our download statistics for Dev Rel team.

Reply:
addressed in latest commit